webservices/ph5tostationxml.py

```diff
@@ -588,9 +571,31 @@ def get_network_date(self):
 if __name__ == '__main__':
 
     args = get_args()
-    args_dict = vars(args)
-    ph5sxml = PH5toStationXML(args_dict)
+    args_dict = vars(args) 
     
+    if args_dict.get('network_list'):
+        args_dict['network_list'] = [x.strip()
+                        for x in args_dict.get('network_list').split(',')]
+    if args_dict.get('reportnum_list'):
+        args_dict['reportnum_list'] = [x.strip()
+                        for x in args_dict.get('reportnum_list').split(',')]
+    if args_dict.get('sta_list'):
+        args_dict['sta_list'] = [x.strip()
+                        for x in args_dict.get('sta_list').split(',')]
+    if args_dict.get('array_list'):
+        args_dict['array_list'] = [x.strip()
+                        for x in args_dict.get('array_list').split(',')]    
+    if args_dict.get('location_list'):
+        args_dict['location_list'] = [x.strip()
+                        for x in args_dict.get('location_list').split(',')]
+    if args_dict.get('channel_list'):
+        args_dict['channel_list'] = [x.strip()
+                        for x in args_dict.get('channel_list').split(',')]
+    if args_dict.get('ph5path'):
+        args_dict['ph5path'] = args_dict.get('ph5path').split(',')
+
+    ph5sxml = PH5toStationXML(args_dict)
+

```

Comment:
Conversions from comma separated strings to Python List objects were moved from spots throughout the PH5toStationXML class into the `__main__` method.